Move static directory to exampleSite

janraasch opened the pull request:
Hi there, first of all thank you for this wonderful theme 👍 💟 .

For my own website (https://www.janraasch.com), I moved the static-directory out of the root of the theme.

Otherwise, those files are served on every website which uses this theme which seems odd...

E.g. a user might not want to serve a my-domain/images/social.jpg with the image https://github.com/escalate/hugo-split-theme/blob/master/static/images/social.jpg

Maybe this is useful for somebody else as well 😸 .

janraasch committed:
Move static directory to exampleSite

Otherwise those files are served on every website which uses this theme which seems odd.

E.g. a user might not want to serve a my-domain/images/social.jpg with the image https://github.com/escalate/hugo-split-theme/blob/master/static/images/social.jpg

escalate commented:
Hello @janraasch,

thank you for your contribution. If we move the sass and css files to another folder the whole theme would not work. If we do this we have to use Hugo Pipelines with an asset directory.
Take a look here: https://gohugo.io/hugo-pipes/introduction/#asset-directory

Do you have time to introduce Hugo pipelines to this theme?

Best regards
Felix

janraasch committed:
Revert "Move static directory to exampleSite"

This reverts commit 0522465f9f73e85cc6555bd0d3c9dc7699abda8d.

janraasch committed:
Move images, videos and favicon to exampleSite

Otherwise those files are served on every website which uses this theme which seems odd.

E.g. a user might not want to serve a my-domain/images/social.jpg with the image https://github.com/escalate/hugo-split-theme/blob/master/static/images/social.jpg

janraasch commented:
Hi @escalate, thank you for your reply. I think I overstepped the mark here a little bit 😜 😺 . The `static/assets` are needed by all sites which use this theme - as you noted correctly. What was bothering me, is that my site was serving the standard `images` and `videos`.

So here's a new commit which only moves those files to the exampleSite: https://github.com/escalate/hugo-split-theme/pull/9/commits/bd246f86b23321233576440b4f187eb7837f8fea

PS:

About moving to `Hugo Pipelines`. I have a theme using `Hugo Pipelines` (https://github.com/janraasch/hugo-product-launch), but I'd suggest to keep it as simple as possible, and currently, it does not seem like you would *really really* need it here.

escalate commented:
LGTM
